bower.json

```diff
@@ -10,7 +10,7 @@
     "purescript-prelude": "^3.1.1",
     "purescript-react": "^v5.1.0",
     "purescript-maps": "^3.6.0",
-    "purescript-react-map-gl": "git://github.com/f-o-a-m/purescript-react-map-gl.git#888d5bbed93e3e2b2c17f0f5ae71facd516ea680"
+    "purescript-web-mercator": "f-o-a-m/purescript-web-mercator#master"
```

Comment:
Let's use a tag or commit hash of ps-web-mercator to avoid getting issues with any future changes of `master`.